jobs/rocoto/waveinit.sh

```diff
@@ -19,6 +21,8 @@ echo "=============== START TO RUN WAVE INIT ==============="
 # Execute the JJOB
 ${HOMEgfs}/jobs/JGLOBAL_WAVE_INIT
 status=$?
-[[ ${status} -ne 0 ]] && exit ${status}
+if [[ ${status} -ne 0 ]]; then
+    exit "${status}"
+fi
 
 exit 0
```

Comment:
The other rocoto job scripts have `exit "${status}"` at the very end. Do we need the `exit 0` or the if-block here?